src/selectors/userContent.js

```diff
@@ -0,0 +1,38 @@
+import { createSelector } from 'reselect'
+import flatten from 'lodash.flatten'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'lodash.flatten'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->